backend/api/controllers/PullRequestController.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const PullRequest = require("../../services/PullRequest");
+
+const PullRequestController = express();
+
+PullRequestController.get("/", async function(req, res) {
```

Comment:
Maybe it will be better to have a `/pulls/:year` endpoint, using the param `:year` to filter by each year hacktoberfest.